ims/einstein/operations.py

```diff
@@ -681,3 +682,26 @@ def run_script(self, img, script):
                                    constants.DEFAULT_SNAPSHOT_NAME)
             self.fs.remove_snapshot(ceph_img_name,
                                     constants.DEFAULT_SNAPSHOT_NAME)
+
+    # TODO: Write tests for this method before merging to main branch
+    @log
+    def get_iscsi_target(self, node_name):
+        try:
+            if not self.is_admin:
+                raise AuthorizationFailedException()
+            provisioned = False
+            provisioned_nodes = self.list_provisioned_nodes()['retval']
+            for node in provisioned_nodes:
+                if node_name == node[0]:
```

Comment:
Wait, why are we returning both image and parent name in list_images? I know that we might have done it for a specific reason but I can't remember of anything. Also, this needs to be separate discussion, let's not block PR on this.